client/app/bundles/HelloWorld/startup/registration.jsx

```diff
@@ -0,0 +1,14 @@
+import ReactOnRails from 'react-on-rails';
+
+import HelloWorld from '../components/HelloWorld';
+
+import Header from '../components/Header';
+
+import ResturantsList from '../components/ResturantsList';
+
+import ResturantItem from '../components/ResturantItem';
+
+// This is how react_on_rails can see the HelloWorld in the browser.
+ReactOnRails.register({
+  HelloWorld, Header, ResturantsList, ResturantItem
```

Comment:
So, question - should the "App" component be for all of the app's pages? how do I control it's body content? should I pass it other components as props to be rendered inside it?